src/jsx.d.ts

```diff
@@ -601,7 +601,7 @@ export namespace JSXInternal {
 		srcSet?: string;
 		start?: number;
 		step?: number | string;
-		style?: any;
+		style?: {[key: string]: string | number};
```

Comment:
kinda wondering if there would be any lib where this would not be enough

Reply:
Would this be a better fit maybe? I think all style values should fit into either `string | number`, but there may be some edge cases (like `grid-template-area`, where you can pass multiple strings)
```suggestion
		style?: {[key: string]: any};
```

Reply:
but even there it would be a string in js? idk

was thinking like one of the css-in-js libs that passes stuff to style might break, but I'm fine with merging it like this and potentially making it broader if anyone runs into issues

Reply:
Ok, why don't I just keep it as is for now, and we can always approach it again if someone finds a lib where it breaks.